frontend/public/components/utils/kebab.tsx

```diff
@@ -256,7 +256,7 @@ const kebabFactory: KebabFactory = {
   Delete: (kind, obj) => ({
     // t('details-page~Delete {{kind}}', {kind: kind.label})
     labelKey: 'details-page~Delete {{kind}}',
-    labelKind: { kind: kind.label },
+    labelKind: { kind: i18next.t(kind.labelKey) },
```

Comment:
We're using an internationalized string as a parameter in another internationalized string. Wondering if this might cause issues. For example, if the non-parameterized part of the string is reactive and the parameter isn't, the user might select a new language and only part of the string would change.

Reply:
You can nest them. We do this elsewhere in the application as well.